src/main/java/TaskType.java

```diff
@@ -0,0 +1,3 @@
+public enum TaskType {
+    TODO,DEADLINE,EVENT
```

Comment:
Add spaces in between the commas for more readable code
```suggestion
    TODO, DEADLINE, EVENT
```